src/main/groovy/pl/wybcz/MicroserviceTemplateBuilder.groovy

```diff
@@ -1,12 +1,14 @@
 package pl.wybcz
 
+import groovy.transform.CompileStatic
 import javaposse.jobdsl.dsl.DslFactory
 
+@CompileStatic
 class MicroserviceTemplateBuilder {
 
-    DslFactory dslFactory
-    PrBuilder prBuilder
-    List<GitProject> projects
+    private DslFactory dslFactory
+    private PrBuilder prBuilder
+    private List<GitProject> projects
```

Comment:
yeah it was planned to make setting of all of that via maps - looks nicer from DSL perspective